node/plugin/cmd/main.go

```diff
@@ -117,6 +124,21 @@ func pluginOps(ctx *cli.Context) {
 		}
 	}
 
+	if config.Operation == plugin.OperationListQuorums {
```

Comment:
why don't we put this block before declaring `sk` and `privateKey`? Then we don't need the block in L77, right?

Reply:
I thought about that but the reason is - this still require `transactor` which requires `ethConfig` which requires `privateKey` - well `privateKey` is just passed as config. One way is I create `ethConfig` without privateKey and if we get to a point where `privateKey` is needed then I can add that config. would that be better?

Reply:
oh but that won't work since `transactor` will already be created by then. I don't think we want to create `transactor` again.